changes/933.feature.md

```diff
@@ -0,0 +1 @@
+Skip scanning wrong named image.
```

Comment:
Add more details like:
```suggestion
When rescanning container image registries, skip misformatted image names and tag names instead of failing the entire process
```